flow/data_pipeline/query.py

```diff
@@ -552,6 +556,22 @@ class QueryStrings(Enum):
         ORDER BY time_seconds_bin ASC
         ;"""
 
+    FACT_VEHICLE_COUNTS_BY_TIME = """
+        SELECT
+            vt.source_id,
+            vt.time_step - FIRST_VALUE(vt.time_step)
+                OVER (PARTITION BY vt.source_id ORDER BY vt.time_step ASC) AS time_step,
```

Comment:
Presto don't support having window functions in a group by column, I think you'll have to group by first and perform window function on the view here

Reply:
ah good catch